pgml-cms/docs/SUMMARY.md

```diff
@@ -79,7 +80,7 @@
 ## Resources
 
 * [FAQs](resources/faqs.md)
-* [Data Storage & Retrieval](resources/data-storage-and-retrieval/README.md)
+* [Data Storage & Retrieval](resources/data-storage-and-retrieval/tabular-data.md)
```

Comment:
This is a dup?

Reply:
Not a dup, README.md is currently empty.